### PR TITLE
K8s: code block spacing fix

### PR DESCRIPTION
**Pull request by kaitlynmichael**
DOC-2151

### CODE REVIEW

File: content/operate/kubernetes/deployment/openshift/openshift-cli.md
Comment (cmilesb) [inline content/operate/kubernetes/deployment/openshift/openshift-cli.md:1]: The note on Line 113 in this file also has a spacing issue.

File: content/embeds/k8s-admission-webhook-cert.md [hunk 1]
```diff
@@ -27,15 +27,15 @@
...
-    ```
+```sh
+cat > modified-webhook.yaml <<EOF
+webhooks:
+- name: redisenterprise.admission.redislabs
+  clientConfig:
+    caBundle: $CERT
+  admissionReviewVersions: ["v1beta1"]
+EOF
+```
```
Comment (dwdougherty) [inline content/embeds/k8s-admission-webhook-cert.md:38]: This item doesn't have the same indentation as the others above it, and it kinda sticks out like a sore thumb. Same for the other page. All the code examples should have uniform indentation.

**Review by cmilesb — CHANGES_REQUESTED** [review]
There's one note that still has a spacing issue. The rest look okay.

**Review by dwdougherty — COMMENTED** [review]
Just one comment.

**Review by cmilesb — APPROVED** [review]
Looks good!

**Pull request merged**